db/migrations/20200202192230_create_games.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+Hanami::Model.migration do
+  change do
+    create_table :games do
+      primary_key :id
+
+      column :title, String, null: false, unique: true, index: true
+      column :us_title, String, null: true, unique: true
+      column :uk_title, String, null: true, unique: true
```

Comment:
Do we definitely want a uniqueness constraint on game names? Is it not conceivable that 2 games could have the same name? Like a remake or something?

Reply:
I don't think this is a problem because the games table only holds the titles. If there's a game with the same name but is for some reason a different version, then that should be handled at the release or publication level.